File: examples/python/testing-patterns/basic-tests/use-fixtures.py
# conftest.py
@pytest.fixture
def valid_order():
    """Provide valid order data."""
    return {
        "order_id": "order-123",
        "customer_id": "customer-456",
        "amount": 100.0,
        "items": [
            {"product_id": "prod-1", "quantity": 2},
            {"product_id": "prod-2", "quantity": 1},
        ],
    }

# test_orders.py
@pytest.mark.durable_execution(handler=handler, lambda_function_name="orders")
def test_order_processing(durable_runner, valid_order):
    """Test order processing with valid data."""
    with durable_runner:
        result = durable_runner.run(input=valid_order, timeout=10)
    
    assert result.status is InvocationStatus.SUCCEEDED

File: examples/java/testing-patterns/basic-tests/use-fixtures.java
class OrderProcessingTest {

    private Map<String, Object> validOrder;

    @BeforeEach
    void setUp() {
        validOrder = Map.of(
            "order_id", "order-123",
            "customer_id", "customer-456",
            "amount", 100.0,
            "items", List.of(
                Map.of("product_id", "prod-1", "quantity", 2),
                Map.of("product_id", "prod-2", "quantity", 1)
            )
        );
    }

    @Test
    void testOrderProcessing() {
        var runner = LocalDurableTestRunner.create(Map.class, new OrderHandler());

        var result = runner.runUntilComplete(validOrder);

        assertEquals(ExecutionStatus.SUCCEEDED, result.getStatus());
    }
}
